test/decl/protocol/req/associated_type_inference_fixed_type.swift

```diff
@@ -41,8 +41,7 @@ protocol P5b: P5a where A == Self {}
 struct S5<X>: P5b {} // OK, A := S5<X>
 
 
-protocol P6 where A == Never { // expected-error {{same-type constraint type 'Never' does not conform to required protocol 'P6'}}
-  // expected-error@+2 {{same-type constraint type 'Never' does not conform to required protocol 'P6'}}
+protocol P6 where A == Never { // expected-error {{no type for 'Self.A' can satisfy both 'Self.A == Never' and 'Self.A : P6'}}
```

Comment:
The new phrasing here sounds like a loss of clarity. Are the old phrasings on their way out?